web/backend/backend.org

```diff
@@ -243,23 +243,24 @@ The core file in this processed folder is  =apisnoop.json=.  This lists all the
 **** Define addEntryToReleasesService
```

Comment:
Not specific to this PR, but have you taken a look at the github rendering?
Do you feel it includes everything you'd like it to show?

Reply:
I haven't reallly considered github rendering for this.  The writing here is originally for one ii/apisnoop developer to another, and so assumed it'd be read within emacs.  This can be adjusted, or exported, though but not sure the priority yet.

Reply:
I think in general we should update our style so that it renders well (on github, gitlab, and in general gollum). While we do flow inside org, I'd like it to be as accessible as possible.